framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/WorkbenchDeploymentImpl.java

```diff
@@ -95,4 +105,42 @@ public String getServiceName() {
             getSecureUrl().ifPresent(RouterUtil::waitForRouter);
         }
     }
+
+    @Override
+    public void changePassword(String newPassword) {
+        deploySecretAppUser(DeploymentConstants.getAppUser(), newPassword);
+    }
+
+    @Override
+    public void changeUsernameAndPassword(String newUsername, String newPassword) {
+        deploySecretAppUser(newUsername, newPassword);
+    }
+
+    private void deploySecretAppUser(String user, String password) {
+        logger.info("Delete old secret '{}'", DeploymentConstants.getAppCredentialsSecretName());
+        getOpenShift().secrets().withName(DeploymentConstants.getAppCredentialsSecretName()).delete();
+        new SimpleWaiter(() -> getOpenShift().getSecret(DeploymentConstants.getAppCredentialsSecretName()) == null).timeout(TimeUnit.MINUTES, 2)
+                                                                                                                   .reason("Waiting for old secret to be deleted.")
+                                                                                                                   .waitFor();
+
+        logger.info("Creating user secret '{}'", DeploymentConstants.getAppCredentialsSecretName());
+        Map<String, String> data = new HashMap<>();
+        data.put(OpenShiftConstants.KIE_ADMIN_USER, user);
+        data.put(OpenShiftConstants.KIE_ADMIN_PWD, password);
+        
+        getProject().createSecret(DeploymentConstants.getAppCredentialsSecretName(), data);
```

Comment:
This is duplicated in OpenShiftScenario.deploySecretAppUser, does it make sense to create a UserSecretDeployer as done for AmqSecretDeployer?

Also, there is already an existing method to replace secrets:

```
project.getOpenShift().secrets().createOrReplaceWithNew(). 
...
```